### PR TITLE
Import result check

### PR DESCRIPTION
**Pull request by kvahed**

### CODE REVIEW

File: tests/simple/simple_import.go [hunk 1]
```diff
@@ -66,6 +70,48 @@ func (t *simpleTest) importDocuments(c *collection) error {
...
+				if created, ok := v["created"]; ok {
+					if cint, ok := created.(float64); ok {
+						if cint != ndocs {
+							if details, ok := v["details"]; ok {
+								t.importCounter.failed++
+								t.reportFailure(
+									test.NewFailure(
+										"Failed to import documents in collection '%s': incomplete import of only %d documents, details: %v",
+										c.name, cint, details))
+								return maskAny(fmt.Errorf("Failed to import documents in collection '%s': incomplete import"))
```
Comment (neunhoef) [inline tests/simple/simple_import.go:84]: This is missing the `c.name` argument.

File: tests/simple/simple_import.go [hunk 2]
```diff
@@ -66,6 +70,48 @@ func (t *simpleTest) importDocuments(c *collection) error {
...
+							} else { // details missing although error
+								t.reportFailure(test.NewFailure(
+									"Failed to import documents in collection '%s': incomplete import of only %d documents no details(!!!)",
+									c.name, cint))
+								return maskAny(fmt.Errorf("Failed to import documents in collection '%s': incomplete import"))
```
Comment (neunhoef) [inline tests/simple/simple_import.go:89]: See above.

**Review by neunhoef — CHANGES_REQUESTED** [review]
Further changes necessary: In `simple.go` we must handle the error case better. At the very least a `continue` is missing when the error occurs. We might have to clean up the data structures in the testagent. Also note that the collection will exist but will contain either none or only some of the planned documents. We might want to forget about the collection in the test agent but leave it in the database for later inspection.

**Comment by kvahed** [comment]
The rest in a diffent PR.

**Review by neunhoef — APPROVED** [review]
LGTM

**Pull request merged**